spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkScan.java

```diff
@@ -734,6 +795,19 @@ private Expression[] expressions(Expression... expressions) {
     return expressions;
   }
 
+  private void checkStatistics(SparkScan scan, long expectedRowCount, boolean expectedColumnStats) {
```

Comment:
when `expectedColumnStats` is false, can we assert the absence of stats

Reply:
Added the check. Thanks